controllers/suite_test.go

```diff
@@ -52,13 +55,19 @@ func TestAPIs(t *testing.T) {
 var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
 
+	// call start or refactor when moving to "normal" testEnv test
```

Comment:
why is start/stopTestEnv needed rather that Before/AfterSuite?

Reply:
I start and stop for every test, see BeforeEach / AfterEach in `controller_test.go`. Lazy way of cleaning test resources after every test and keeping the test code nearer to the original version, where the old fake client was re-initialized for every test